### PR TITLE
Adds third batch of RNs for 4.14

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.14

Issue:
No issue. Bug doc text.

Link to docs preview:
https://66435--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bug-fixes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1898,7 +1904,7 @@ As a workaround, you can delete all pods in the `openshift-network-node-identify
 +
 For more information about early access, xref:../updating/understanding_updates/understanding-update-channels-release.adoc#candidate-version-channel_understanding-update-channels-releases[candidate-4.14 channel].
 
-* Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a Daemon Set, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
+* Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a DaemonSet, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
```
Comment (mburke5678) [inline release_notes/ocp-4-14-release-notes.adoc:1907]: s/DaemonSet/daemon set or `DaemonSet` object

```suggestion
* Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a DaemonSet, the `/etc/cni/tuning/allowlist.conf` file on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
```

Comment (mburke5678) [reply release_notes/ocp-4-14-release-notes.adoc:1907]: s/DaemonSet/daemon set or `DaemonSet` object

**Comment by mburke5678** [comment]
@stevsmit A couple of nit. Otherwise LGTM

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (stevsmit) [reply release_notes/ocp-4-14-release-notes.adoc:1907]: Weird. I had fixed DaemonSet on my side but I guess I didn't push it. Thanks!

**Pull request merged**